tests/routes/get_recommendations_test.py

```diff
@@ -842,3 +844,47 @@ def when_searching_by_keywords_offer_with_matching_pattern_in_name_return_firsts
             assert response.json[2]['offer']['name'] == 'Modern Tango'
             assert response.json[3]['offer']['name'] == 'Training'
             assert response.json[4]['offer']['name'] == 'Tango Plus'
+
+        @clean_database
+        def when_searching_by_keywords_offer_with_search_feature_deactivated(self, app):
+            deactivate_feature(FeatureToggle.FULL_OFFERS_SEARCH_WITH_OFFERER_AND_VENUE)
+            user = create_user(email='test@email.com')
```

Comment:
```suggestion
            user = create_user(email='test@example.com')
```
Et si tu pouvais faire les autres, ça serait tiptopmegagroove.